topics/fair/tutorials/fair-ego/tutorial.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: tutorial_hands_on
+title: FAIR and RDM from a selfish perspective
+abbreviations:
+  FAIR: Findable, Accessible, Interoperable, Reusable
+  GTN: Galaxy Training Network
+  RDM: Research Data Management
+  DPM: Data Management Plan
+level: Introductory
+zenodo_link: ''
+questions:
+- Why should other researchers adhere to FAIR principles?
+- How can I profit from good RDM?
+objectives:
+- Learn about the benefits of good data management
+time_estimation: "30M"
+key_points:
+- RDM can lead save time and money.
+tags:
+- fair
+- gtn
+- rdm
+contributions:
+  authorship:
+    - korneelhens
+    - nomadscientist
+  funding:
+    - elixir-fair-data
+subtopic: fair-data
+
+requirements:
+  - type: "internal"
+    topic_name: fair
+    tutorials:
+      - fair-intro
+   
+draft: true
+
+---
+
+
+# Introduction
+
+**You** are obviously too important to bother with this {RDM} nonsense. Just copy the {DMP} from a previous grant application and stop wasting time.
+However, there might be some value in persuading your colleagues to do their {RDM} properly and to adhere to {FAIR} standards. In this tutorial, we explore the purely selfish reasons for encouraging good data management practices based on some real-life examples.
+
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Excercise 1: Find the experiment
+
+## The Problem
+
+Imagine the following situation: An undergrad student wants to do a lab-based project with you. This is great because your post-doc has recently left the lab and there are several promising leads from their work that can be followed up. You propose the following topic to the student: 
+
+**Effect of mutations in the gene CHES-I-like on feeding behaviour in _Drosophila melanogaster_**
```

Comment:
```suggestion
> Effect of mutations in the gene CHES-I-like on feeding behaviour in _Drosophila melanogaster_
{: .quote}
```

we try and avoid bold text in these situations so maybe a quote gives the same feeling of "this sets apart the text"